nemo/collections/asr/models/configs/classification_models_config.py

```diff
@@ -53,6 +53,10 @@ class EncDecClassificationDatasetConfig(nemo.core.classes.dataset.DatasetConfig)
     normalize_audio: bool = False
     is_regression_task: bool = False
 
+    # bucketing params
+    bucketing_strategy: str = "synced_randomized"
```

Comment:
Can you add bucketing strategy and bucketing batch size in the configs as well ?

Reply:
`bucketing_strategy` and `bucketing_batch_size` were already present in the marblenet and matchboxnet example configs. According to git blame, they were added as part of the PR that added randomized bucketing (https://github.com/NVIDIA/NeMo/pull/3445).

Reply:
Perfect. Thanks